src/languages/java.md

```diff
@@ -1,3 +1,101 @@
 # Java
 
-Java support is in Alpha and not generally available. Contact sales [https://platform.sh/contact](https://platform.sh/contact) if you are interested in running Java based applications on Platform.sh.
+Java is a general-purpose programming language, and one of the most popular in the world today. Platform.sh supports Java runtimes that can be used with build management tools such as Gradle, Maven, and Ant.
+
+
+## Supported versions
+
+* 8
+* 11
+* 12
+
+
+To select a Java version, specify a `type` such as `java:11`:
```

Comment:
Somewhere in here we should specify that we run OpenJDK specifically.  I'm... not sure of the best way to phrase that.

Reply:
Just put it as a line above the version numbers.

Reply:
Done